cmake/protobuf-lite.pc.cmake

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@
```

Comment:
this two lines still looks fishy.
Somehow, `@CMAKE_INSTALL_LIBDIR@` and `@CMAKE_INSTALL_INCLUDEDIR@` are still empty sometimes, maybe configure it later (in `install.cmake`?)